pkg/appfile/parser.go

```diff
@@ -219,7 +219,7 @@ func (p *Parser) parseTrait(ctx context.Context, name string, properties map[str
 
 // GenerateApplicationConfiguration converts an appFile to applicationConfig & Components
 func (p *Parser) GenerateApplicationConfiguration(app *Appfile, ns string) (*v1alpha2.ApplicationConfiguration,
```

Comment:
checking out side by the info AppConfig and Component makes the code more clear.

Please don't add a return value everywhere.